### ISSUE TITLE
Update documentation about add index.md files inside children.

### ISSUE THREAD

**Issue by Barbapapazes** [nuxt/content#2204]
### Environment

/

### Reproduction

/

### Describe the bug

If you add an `index.md` file inside a folder, it will be added inside the children from the navigation part.

**Comment by christophermiles** [comment]
Agree, I don't understand what the convention is meant to be here. I've followed the example from the documentation and now I have a bizarre navigation tree that has each page being its own child.
